pages/common/git-gui.md

```diff
@@ -0,0 +1,37 @@
+# git gui
+
+> A GUI for making new commits, amending existing ones, creating branches, performing local merges, and fetching/pushing to remote repositories.
```

Comment:
Maybe we could shorten or break the description a bit.

```suggestion
> A GUI for making new and amending commits, creating branches, performing local merges, and fetching/pushing to remote repositories.
```
(or)

```suggestion
> A GUI for Git to manage branches, commits, and remotes.
```

Feel free to reword it differently too.